styles.css

```diff
@@ -116,6 +124,395 @@ li {
   text-decoration: none;
 }
 
+/* start js */
+
+.work-container {
+  border: 1px solid #cbcdd1;
+  display: grid;
+  grid-template-columns: auto;
+  justify-content: center;
+  align-content: center;
+  text-align: left;
+  width: 95%;
+  margin-top: 15vh;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 50px;
+  gap: 12px;
+  background: #fff;
+  border-radius: 16px;
+}
+
+.modal {
+  position: fixed;
+  display: none;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  height: 100%;
+  width: 100%;
+  overflow: auto;
+  align-items: center;
+  background-color: rgba(0, 0, 0, 0.5);
+}
+
+.work-content-modal {
+  border: 1px solid #cbcdd1;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: left;
+  width: 95%;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 30px;
+  padding: 20px;
+  gap: 12px;
+  background: #fff;
+  border-radius: 16px;
+  -webkit-backdrop-filter: blur(8px);
+  backdrop-filter: blur(8px);
+}
+
```

Comment:
- Kindly align the grey line to the center as it is on the Figma design

![image](https://user-images.githubusercontent.com/93630700/223740623-41bffcda-da53-4899-ae27-11d49795edfb.png)